web/templates/datasets/dataset_wizard_form.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'layout.html' %}
+{% load static %}
+{% block head_end %}
+    {{ wizard.form.media }}
+    <link rel="stylesheet" href="{% static 'css/dataset_wizard.css' %}">
+{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="jumbotron mt-5 py-2 pt-4 px-4 w-100">
+            <ul class="step-wizard-list">
+                {% for step, step_verbose_name, skipped in steps_verbose_data %}
+                    <li class="step-wizard-item {% if wizard.steps.current == step %}current-item{% endif %}">
+                        <span class="progress-count {% if skipped %}skipped{% endif %}">{{ forloop.counter }}</span>
+                        <span class="progress-label">{{ step_verbose_name }}</span>
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+
+        <div class="col-md-12 card">
+            <div class="d-flex justify-content-between mt-3">
+                <a href="{% url 'wizard' %}?reset=true"
+                   class="btn btn-outline-secondary btn-raised float-right">
+                    <i class="material-icons" style="line-height:1.5;">refresh</i>
+                </a>
```

Comment:
The reset button does not fully reset the form, as the dataset that was created is not deleted.